warehouse/internal/loadfiles/loadfiles.go

```diff
@@ -237,13 +234,11 @@ func (lf *LoadFileGenerator) createFromStaging(ctx context.Context, job model.Up
 		}
 		// set messages to nil to release mem allocated
 		messages = nil
-		batchStartIdx := i
-		batchEndIdx := j
-
+		startId := chunk[0].ID
+		endId := chunk[len(chunk)-1].ID
 		g.Go(func() error {
 			responses := <-ch
-			lf.Logger.Infof("[WH]: Received responses for staging files %d:%d for %s:%s from PgNotifier", toProcessStagingFiles[batchStartIdx].ID, toProcessStagingFiles[batchEndIdx-1].ID, destType, destID)
-
+			lf.Logger.Infof("[WH]: Received responses for staging files %d:%d for %s:%s from PgNotifier", startId, endId, destType, destID)
```

Comment:
we can convert this to structured log:
```suggestion
			lf.Logger.Infow("Received responses for staging files %d:%d for %s:%s from PgNotifier", 
			"startId", startId, 
			"endID", endId,
			 logfields.DestType, destType,
			 logfields.DestID, destID,
	)
```